votes/2022-07-06.json

```
@@ -0,0 +1,52 @@
+{
+  "description": "Nomination of Geoffrey Booth as a TSC member",
+  "outcome": {
+    "winner(s)": [
+      "Yes"
+    ]
+  },
+  "votes": {
+    "aduh95": {
+      "Yes": 1
+    },
```

nit: on other "binary" votes, we were using another format (e.g. https://github.com/nodejs/TSC/blob/main/votes/2022-04-11-1.json). Not that it matters though.
```suggestion
  "outcome": "Yes",
  "votes": {
    "aduh95": 1,
```